src/utils/json.cpp

```diff
@@ -88,6 +90,22 @@ aos::RetWithError<Poco::Dynamic::Var> ParseJson(std::istream& in) noexcept
     }
 }
 
+Error WriteJsonToFile(const Poco::JSON::Object::Ptr& json, const std::string& path)
+{
+    std::ofstream file(path);
+    if (!file.is_open()) {
+        return ErrorEnum::eFailed;
+    }
+
+    try {
+        Poco::JSON::Stringifier::stringify(json, file);
+    } catch (const std::exception& e) {
+        return ErrorEnum::eRuntime;
```

Comment:
It's worth adding log with error that will describe the reason of the failure

Reply:
Done